Explorer/Assets/DCL/InWorldCamera/InWorldCamera/ScreenshotMetadataBuilder.cs

```diff
@@ -83,7 +83,7 @@ public async UniTask BuildAsync(CancellationToken ct)
 
             PlacesData.PlaceInfo? placeInfo = await placesAPIService.GetPlaceAsync(at, ct);
 
-            return (placeInfo?.title ?? "Unknown place", placeInfo.id);
+            return placeInfo == null ? ("Unknown place", "Unknown place") : (placeInfo.title, placeInfo.id);
```

Comment:
I guess a micro optimization could be having the "Unknown place" (or even the whole `("Unknown place", "Unknown place")`) in a `const`, although maybe the compiler is already smart enought to re-use it the way you already have it today.